modernism/static/_js/modernism.js

```diff
@@ -126,6 +126,24 @@ const setModalImage = (direction) => {
 
 }
 
+//Get the button
```

Comment:
Please remove comments here, as it is clear by reading the code what happens.

Reply:
I will removed it.

Reply:
And thank you for the detailed description of the improvement points.